.github/resources/config-for-integration-test.json

```diff
@@ -53,6 +53,8 @@
             "port": 32331,
             "flags": [
               "--rpc-cors=all",
+              "--unsafe-ws-external",
```

Comment:
Did you find the reason for this change?

Does this affect only CI or will it affect community collators when they run the node?

Reply:
I didn't run it on AWS. So next step, we need to deploy the client to cloud, to find out whether this flag will affect us and node runners.

Reply:
Now it's working without this `unsafe-xxx` flag locally, weird.

Reply:
See this issue https://github.com/paritytech/polkadot/issues/5647